### ISSUE TITLE
WebHost (Minimal) HTTP Error 404.0 - Not Found

### ISSUE THREAD

**Issue by farzad99** [IdentityServer/IdentityServer3#2282]
I'm running the WebHost (minimal) sample and the Clients sample at the same time. Any client that I use to login I get the HTTP Error 404.0 - Not Found. I can run the SelfHost (minimal) which is a console application with the Clients sample and everything works. Could somebody please tell me what I need to do to make the WebHost (minimal) work for me?

**Comment by brockallen** [comment]
SSL? Did you use `Map` and are browsing to that path?

**Comment by farzad99** [comment]
Thank you for your response. So here's what I did I downloaded the samples. I ran the WebHost (minimal) and I saw the "Welcome to IdentityServer3 (build 2.0.0.174)" page at

 https://localhost:44333/

Next I ran the clients sample and used the "JavaScript Implicit Client - Simple" with the url of 

http://localhost:37045/index.html

I clicked the "Get Token" button of the above page and I got the following error:

HTTP Error 404.0 - Not Found

The Requested URL in the error page was:

https://localhost:44333/core/connect/authorize?client_id=implicitclient&redirect_uri=http://localhost:37045/index.html&response_type=token&scope=write&state=14499769857470.965534688195719

So the IdentityServer is using SSL and if I click the Discovery Document link it displays all the all end points. I'm not sure what changes I need to make to the WebHost (minimal) sample to get it to work. The SelfHost (minimal) sample works with the Clients sample but the WebHost (minimal) sample doesn't work with the Clients sample. 

I would appreciate your help.
